.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+/src/main/resources/log4j2.xml
```

Comment:
The path for `log4j2.xml` should be adjusted to ignore it in any subdirectory. Consider using a pattern like `**/log4j2.xml` to ensure it is ignored regardless of its location within the project.